pkg/hubble/exporter/config.go

```diff
@@ -241,3 +253,60 @@ func (f FieldMask) equals(other FieldMask) bool {
 	slices.Sort(ys)
 	return reflect.DeepEqual(xs, ys)
 }
+
+// FieldAggregate is a slice of fields that are used for flowlog aggregation.
+type FieldAggregate []string
+
+func (f FieldAggregate) equals(other FieldAggregate) bool {
+	xs := make([]string, len(f))
+	ys := make([]string, len(other))
+	copy(xs, f)
+	copy(ys, other)
+	slices.Sort(xs)
+	slices.Sort(ys)
+	return reflect.DeepEqual(xs, ys)
+}
+
+// Duration is a wrapper around time.Duration that can unmarshal from YAML strings.
+type Duration time.Duration
```

Comment:
I'm confused by this. I'm pretty sure this isn't needed and unmarshalling from a string to a time.Duration should work with the `go.yaml.in/yaml/v3` package out of the box: https://github.com/yaml/go-yaml/blob/779357c6448d90a4e0ccd27269bc6cfc789d532e/decode.go#L658-L666

Reply:
I just checked this. The lib used is `sigs.k8s.io/yaml` and if we switch to `go.yaml.in/yaml/v3` it fails to parse snake case filters. In the current scenario i used a wrapper to be able to parse human readable format like `0s` otherwise for its expecting the durations in nanoseconds.